homework-submission/todo-app/src/App.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import './App.css';
+import Todos from './Components/Todos';
+import TodoItems from './Components/TodoItems.json';
+
+class App extends Component {
+  state = {
+    TodoItems
+  }
+
+  changeTodo = id => {
+
+    let todos = [...this.state.TodoItems];
+
+    todos.forEach(item => {
+      if (item.id === id) {
```

Comment:
it should be const and don't copy by reference but value so you should make a deep copy

https://www.youtube.com/watch?v=EeZBKv34mm4